src/PythConnection.ts

```diff
@@ -84,6 +95,26 @@ export class PythConnection {
     }
   }
 
+  private async handleStalePrice() {
```

Comment:
imo invoking the callback when prices go stale is not necessary. Users of this interface know that they are getting pushed price updates, and the resulting updates may stop at any time for various reasons (e.g., connection disconnects). They will already have to write code to check when the prices were last updated before using them. I don't think this code helps them much.

Reply:
@Reisen what do you think?

Reply:
If you think the same way I can delete it.

I agree with you Jayant, but on the other hand I don't know what users expect. I believe their expectation is more around connection issues, not this type of event. 
If they are expecting callback to be fired when anything happens to a price I can imagine price going stale be an event.

Reply:
One downside was that it was needed to make a request to get the current slot as onSlotChange only gives latest created slot and does not work with commitment.

Overall, to keep things simple and be more consistent with other clients, removed the callback.